.github/workflows/actions.yml

```diff
@@ -0,0 +1,69 @@
+name: Clerk work
+
+on:
+    schedule:  # Daily schedule.
+        - cron: 0 0 * * *
+
+    workflow_dispatch: {}  # For manual switch in actions page.
+
+    push:  # Run on update to py files or this file on the main branch.
+        branches:
+            - main
+        paths:
+            - "**.py"
+            - ".github/workflows/action.yml"
+# End of on.
+
+
+jobs:
+    scheduled:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout
+                uses: actions/checkout@v3
+
+            - name: Setup Python
+                uses: actions/setup-python@v4
+
+            # https://stackoverflow.com/a/62639424
+            - name: Cache virtual environment
+                uses: actions/cache@v3
+                id: cache-venv
+                with:
+                    path: ~/venv  # Change SOME_STRING to invalidate cache.
+                    key: ${{ runner.os }}-venv-${{ secrets.SOME_STRING }}-${{ hashFiles('**/requirements.txt') }}
+                    restore-keys: |
+                        ${{ runner.os }}-venv-${{ secrets.SOME_STRING }}-
+
+            - name: Make virtual environment
+                if: steps.cache-venv.outputs.cache-hit != 'true'
+                run: |
+                    python3 -m venv ~/venv
+                    . ~/venv/bin/activate
+                    pip install -r requirements.txt
+
+            # Our main clerk work is here.
+
+            - name: Run update script
+                run: |
+                    . ~/venv/bin/activate
+                    python3 fetch_latest.py
+                env:
+                    TRANSIFEX_API_TOKEN: ${{ secrets.TRANSIFEX_API_TOKEN }}
+
+            - name: Commit changes (if any)
+                run: |
+                    if [[ $(git status -s) ]]; then
+                        git config user.name github-actions
+                        git config user.email github-actions[bot]@users.noreply.github.com
```

Comment:
Let's use `git config user.email 41898282+github-actions[bot]@users.noreply.github.com` used so that we get the bot icon as well:

https://github.com/orgs/community/discussions/26560

Reply:
The email used also shows it (that's why I used it).

Reply:
ah, wonderful! good to know